book/src/nostr/04-nip-utils.md

```diff
@@ -0,0 +1,26 @@
+## NIP-Specific Utilities
+
+The [Event](https://docs.rs/nostr/latest/nostr/event/struct.Event.html) struct represents the structure for an event in Nostr. Many of the NIPs define specific `content` and `tags` that are required to correctly represent a kind. The `nostr` crate ships with a set of NIP-specific utilities for working with certain event kinds.
+
+### Metadata (NIP-01)
+
+<custom-tabs category="lang">
+
+<div slot="title">Rust</div>
```

Comment:
Same here: can you add the tabs for all the supported languages, leaving the `TODO` where you not add the examples?